src/supported-operations/running-monitoring-operations.md

```diff
@@ -0,0 +1,94 @@
+# Running and monitoring operations
+  
+## Requirements
+* The progress of each operation request must be logged in a journal,
+  with timestamped entries of when the operation has been 
+  scheduled, launched, cancelled, completed or aborted.
+* Each command execution must be recorded in a log
+  accessible to the device owner and the remote operator.
+  This log contains the full command path and arguments,
+  plus the recorded standard output and error of the process.
+* The device owner should be able to define constraints
+  when a command can be launched / stopped:
+  - Can two instances of the same operation run concurrently?
+  - Can the command run concurrently with another?
+  - Has the process to be detached from the agent daemon?
+  - Has an operation to be aborted on agent restart?
+
+## Proposed design
+* The operations are defined by the configuration files in [`/etc/tedge/operations`](./principles.md).
+* The cloud mappers trigger the operations.
+  * This is done on reception of operation requests from the cloud:
+    each operation configuration defines the topic to listen, and the message-pattern to match.
+  * The responsibility of the mapper is only to translate cloud specific messages into thin-edge requests.
+    When an operation request is received, the mapper combines the request and the operation configuration
```

Comment:
Does this mean that the types of supported operations are already sent to the cloud when mapper is started?.

Reply:
Yes, the mapper notify the cloud end-point on start.